direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaAccessor.java

```diff
@@ -211,15 +212,13 @@ private void configure(Map<String, Object> cfg) {
         getUri());
   }
 
-  @SuppressWarnings("unchecked")
   Properties createProps() {
     Properties props = new Properties();
     for (Map.Entry<String, Object> e : cfg.entrySet()) {
-      if (e.getKey().startsWith(WRITER_CONFIG_PREFIX)) {
-        props.put(e.getKey().substring(PRODUCE_CONFIG_PREFIX_LENGTH), e.getValue().toString());
+      if (e.getKey().startsWith(KAFKA_CONFIG_PREFIX)) {
+        props.put(e.getKey().substring(KAFKA_CONFIG_PREFIX.length()), e.getValue().toString());
       }
     }
-    props.put(MAX_POLL_RECORDS, maxPollRecords);
```

Comment:
this options is already passed in the loop above, but with properly removed prefix